eve/main.yml

```diff
@@ -143,3 +144,70 @@ stages:
             echo "%(secret:ssh_pub_keys)s" >> ~/.ssh/authorized_keys &&
             sleep 3600
           doStepIf: false
+
+  multiple-nodes:
+    worker:
+      type: openstack
+      image: CentOS 7 (PVHVM)
+      flavor: m1.medium
+      path: eve/workers/openstack-multiple-nodes
+    steps:
+      - Git: *git_pull
+      - ShellCommand:
+          name: Init terraform
+          command: "terraform init"
+          workdir: "build/eve/workers/openstack-multiple-nodes/terraform/"
+          haltOnFailure: true
+      - ShellCommand:
```

Comment:
Yeah it's not really mandatory but it can help separate syntax errors from deployment errors. Since the step is executed in less than 1sec, it's not really an issue.